src/nvim/highlight.c

```diff
@@ -847,6 +848,13 @@ HlAttrs dict2hlattrs(Dict(highlight) *dict, bool use_rgb, int *link_id, Error *e
     return hlattrs;
   }
 
+  if (HAS_KEY(dict->blend)) {
+    blend = object_to_color(dict->blend, "blend", true, err);
```

Comment:
What does it mean for the blend key to be a string like `#ff0000` or `Red`? Put another way, why use `object_to_color` here rather than just enforce an integer? Shouldn't blend just be a number between 0 and 100?

Reply:
Because I'm lazy and like to blindly copy and paste stuff. 😕 

I guess the test passing caused me to overlook this.

... I'll fix.